server/src/UseCases/Games/CreateGame/CreateGameCommandHandler.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System.Threading;
+using System.Threading.Tasks;
+using BGS.ApplicationCore.Entities;
+using BGS.SharedKernel;
+using BGS.UseCases.Common.Result;
+using MediatR;
+
+namespace BGS.UseCases.Games.CreateGame;
+
+public class CreateGameCommandHandler : IRequestHandler<CreateGameCommand, Result>
+{
+    private readonly IRepository<Game> _gameRepository;
+
+    public CreateGameCommandHandler(IRepository<Game> gameRepository)
+    {
+        _gameRepository = gameRepository;
+    }
+
+    public async Task<Result> Handle(CreateGameCommand request, CancellationToken cancellationToken)
+    {
+        var newGame = new Game
+        {
+            Name = request.GameName,
+            Key = request.GameName.Replace(" ", ""),
```

Comment:
The `Key` is used to determine the scoring strategy.
When a user creates a new game, after trying to calculate the score, he can't do it because there will be no strategy